upgrading/automated_upgrades.adoc

```diff
@@ -735,6 +735,71 @@ upgrade the component.
 To upgrade an existing EFK logging stack deployment, you review your parameters
 and run the *_openshift-logging/config.yml_* playbook.
 
+The EFK upgrade also upgrades Elasticsearch from the 2.x version to the 5.x version. For important information on changes in Elasticsearch 5.x, you should review the link:https://www.elastic.co/guide/en/elasticsearch/reference/5.5/breaking-changes-5.0.html[Elasticsearch breaking changes].
+
+It is important to note that Elasticsearch 5.x has some significant changes to the index structures. Previously, Elasticsearch permitted a dot character, `.`, in field names.
+In 5.x, Elasticsearch interprets any dot in an Elasticsearch field name as nested structure. If you have a field with a dot, the string after the dot is interpreted as the type of field, leading to mapping conflicts during the upgrade. 
+
+To help identify potential conflicts, {product-title} provides a script that examines your Elasticsearch fields to determine if any fields contain a dot in the name.
+
+For example, the following fields were allowed in Elasticsearch 2.x:
+
+----
+{
+  "field": 123  // "field" is of type number
+}
+
+// Any dot in field name is treated as any other valid character in the field name.
+// It is just part of the field name.
+{
+  "field.name": "Bob"  // "field.name" is of type String 
+----
+
+In Elasticsearch 5.x the `field` string would become the field and the `name` string would become a type for the field: 
+
+----
+{
+  "field": 123  // "field" is of type number
+}
+
+// Any dot in field name is always interpreted as nested structure.
+{
+  "field" : {  // "field" is of type Object
+    "name": "Bob"  // "name" is of type String
+  }
+}
+----
+
+Upgrading in this case would result in the `field` field having two different types, which is not permitted. 
+
+If you need to keep these conflicting indices, you need to reindex the data and change the documents to get rid of conflicting data structure. For more information, see link:https://www.elastic.co/guide/en/elasticsearch/reference/2.4/dots-in-names.html#_upgrading_fields_with_dots_to_5_x[Upgrading fields with dots to 5.x]. 
```

Comment:
It's okay to use `5.x` here because you're referencing a link that is titled `5.x`. Hope this makes sense!